app/controllers/state_file/questions/nj_year_of_death_controller.rb

```diff
@@ -0,0 +1,10 @@
+module StateFile
+  class Questions::NjYearOfDeathController < Questions::QuestionsController
+    include ReturnToReviewConcern
+
+    def self.show?(intake)
+      intake.filing_status_qw?
+    end
+
+  end
+end
```

Comment:
[dust] - this is only 10 lines, but the tests are 87 lines, and it's almost identical to the NC version... is there a way we can reuse the code for our purposes to be more DRY? It's not so much code that I'd consider it to be a boulder, but thinking it'll likely be easier to maintain (and could always break back out if functionality diverges in the future). Thoughts?

Reply:
I generally agree that it feels silly to have these two identical files, but the two things holding me back from DRYing it out in this case are 1) I generally don't think a refactor is worth the effort until the third repetition, especially for something with low complexity like this 2) this PR is necessary for accurate returns and it'll be harder to get PRs through for the next week